### PR TITLE
Add support for Ontology transactions Fixes #94

### LINKED ISSUE #94: Add support for Ontology transactions
`/transactions/AY6pRxUZKopz3rdxwy4yReeZgfBGA2ue48` - regular ONT transfers
`/transaction/AY6pRxUZKopz3rdxwy4yReeZgfBGA2ue48?token=ong` - should return token transfers for ong

### PR DESCRIPTION
**Pull request by ArtemGoryunov**
Please follow this checklist:

 - `platform/mycoin`
   - [x] `model.go`: Platform-specific models
   - [x] `client.go`: Platform getter methods (API, RPC)
   - [x] `api.go`:
     - Gin route: _GET /mycoin/txs_
     - Getting blockchain info
     - Normalizing platform-specific
       models to BlockAtlas model
   - [x] `api_test.go`
     - Unit tests
     - Check if normalization from
       response body matches expected model
 - [x] `test/main.go`:
   Add example address for integration test
 - [x] `loaders.go`: Add route
 - [x] `config.yml`: Add default config
   (comment out if no public endpoint)
 - [x] `config.go`: Add default config

### CODE REVIEW

**Comment by vikmeup** [comment]
Tests? for both ont and ong

File: platform/ontology/client.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		page,
+	)
+
+	res, err := c.HTTPClient.Get(uri)
+	if err != nil {
+		logrus.WithError(err).Errorf("Ontology: Failed to get transactions for address %s for asset %s", address, assetName)
+	}
+	defer res.Body.Close()
+
+	print(res)
```
Comment (vikmeup) [inline platform/ontology/client.go]: can you remove `print(res)` ?

File: platform/ontology/api.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+		ID: srcTx.TxnHash,
+		Coin: coin.ONT,
+		Fee: models.Amount(fee),
+		Date:  srcTx.TxnTime,
+		Block: srcTx.Height,
+		Status: status,
+	}
+
+	// Condition for transfer ONT
+	if assetName == "ont" {
```
Comment (vikmeup) [inline platform/ontology/api.go]: where would you get `ont` value from? user will be requesting `/transactions/:address`

File: platform/ontology/api.go [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+		i := strings.IndexRune(transfer.Amount, '.')
```
Comment (vikmeup) [inline platform/ontology/api.go]: what does this do: `strings.IndexRune(transfer.Amount, '.')` ?

File: platform/ontology/api.go (hunk 2 above)
Comment (ArtemGoryunov) [reply platform/ontology/api.go]: it's here 
https://github.com/TrustWallet/blockatlas/blob/1cef2ded64f0d831ff34a433e4986c629d332b98/platform/ontology/api.go#L31

Comment (vikmeup) [reply platform/ontology/api.go]: I see now, can you have it as constant variables? because it's reused

File: platform/ontology/api.go (hunk 3 above)
Comment (ArtemGoryunov) [reply platform/ontology/api.go]: API returns amount value for ONT/ONG as string like `2.000000000`
ONT is a whole number and `strings.IndexRune(transfer.Amount, '.')` takes whole part.

File: platform/ontology/client.go (hunk 1 above)
Comment (ArtemGoryunov) [reply platform/ontology/client.go]: done

File: platform/ontology/api.go (hunk 2 above)
Comment (ArtemGoryunov) [reply platform/ontology/api.go]: Done

File: platform/ontology/api.go [hunk 4]
```diff
@@ -16,6 +16,8 @@ var client = Client{
 }
 
 const GovernanceContract = "AFmseVrdL9f9oyCzZefL9tG6UbviEH9ugK"
+const ONTAssetName = "ont"
+const ONGAssetName = "ong"
```
Comment (vikmeup) [inline platform/ontology/api.go]: check this file, that's how you specify consts: https://golang.org/src/math/const.go

**Pull request merged**